e-commerce/README.md

```diff
@@ -3,7 +3,7 @@
 This demo is a pair of [DBOS](https://github.com/dbos-inc/dbos-sdk) based systems demonstrating an 
 e-commerce scenario with separate apps for the online shop and the payment provider.
 
-## Demo Setup
+## Demo Setup (local)
 
 This demo requires Node 20.x or later and a PostgreSQL compatible database.
 The demo includes a script to configure a PostgreSQL Docker container on your behalf.
```

Comment:
The docker script is for Postgres 15.4--should we update it to 16.1?

Reply:
Will do that and add a PGPORT override.